mink/lie/se3.py

```diff
@@ -215,6 +217,58 @@ def ljacinv(cls, other: np.ndarray) -> np.ndarray:
         return np.block([[J_inv, -J_inv @ Q @ J_inv], [O, J_inv]])
 
 
+def interpolate_se3(
```

Comment:
If it does not make sense for this function to be in this file, I'm happy to move it elsewhere - let me know what's best!

Reply:
marking this as resolved since we are moving over to an internal implementation as discussed below